cardano-cli/src/Cardano/CLI/Shelley/Key.hs

```diff
@@ -392,6 +392,7 @@ data PaymentVerifier
 data StakeVerifier
   = StakeVerifierKey (VerificationKeyOrFile StakeKey)
   | StakeVerifierScriptFile ScriptFile
+  | StakeVerifierAddress StakeAddress
```

Comment:
:+1: This is much cleaner now.

Reply:
I was not sure all components using StakeVerifier will accept StakeAddress, so I started with ad hoc solution, and then assured I can modify this type.